theme-engine.lua

```diff
@@ -0,0 +1,36 @@
+local HttpService = game:GetService("HttpService")
+
+local themeEngine = {}
+
+function themeEngine.IsUserIsHavingThemeFile()
+    if isFile('Hydroxide/theme.json') then 
+        return true
+    elseif readfile('Hydroxide/theme.json') == '' then -- Hey it's tard proof!
+        writefile( -- Will use hexadecimal color value instead of RGB
+        'theme.json',
+        [[
+{
+    "Icon": {},
+    "SyntaxHighlighting": {},
+    "InterfaceColor": {},
+    "Text": {},
+    "Tweening": {}
+}
+]]
+)    elseif not isFile('Hydroxide/theme.json') then
+        writefile( -- Will use hexadecimal color value instead of RGB
+            'theme.json',
+            [[
+{
+    "Icon": {},
+    "SyntaxHighlighting": {},
+    "InterfaceColor": {},
+    "Text": {},
+    "Tweening": {}
+}
+]]
+)
+    end
+end
+
+return themeEngine
```

Comment:
The code style on this file is really inconsistent compared to the rest of the repository, which uses `snake_case`.

Reply:
the time has come to use snake_case 😥